frontend/src/services/campaign.service.ts

```diff
@@ -14,17 +14,27 @@ function getSentAt(jobs: Array<{ sent_at: Date }>): Date {
   return jobsSentAt[0]
 }
 
-export async function getCampaigns(): Promise<Array<Campaign>> {
-  return axios.get('/campaigns').then((response) => {
-    const campaigns: Campaign[] = response.data.map((data: any) => {
+export async function getCampaigns(params: {
+  offset: number
+  limit: number
+}): Promise<{
+  campaigns: Array<Campaign>
+  totalCount: number
+}> {
+  return axios.get('/campaigns', { params }).then((response) => {
+    const campaigns: Campaign[] = response.data.campaigns.map((data: any) => {
```

Comment:
Would you mind de-structuring response.data? I think we do that for most of the api calls. 
`const {campaigns, total_count} = response.data`

Reply:
Done!